packages/patrol_cli/lib/src/commands/dart_define_utils.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:patrol_cli/src/dart_defines_reader.dart';
+
+/// Merge two map with overriding values for duplicated keys, by [dartDefines] values.
+Map<String, String> mergeKeys({
+  required Map<String, dynamic> json,
+  required Map<String, dynamic> dartDefines,
+}) {
+  final modified = Map<String, String>.from(json);
+
+  if (dartDefines.isNotEmpty) {
+    dartDefines.forEach((key, value) {
+      modified[key] = value as String;
```

Comment:
@pdenert 
I tried this change in our project and realized that not everything is in the form of strings, so the code crashes here - I suspect that other projects may have a similar problem

Reply:
You're right. Can you create issue for that? We will try to address this ASAP

Reply:
Done https://github.com/leancodepl/patrol/issues/2242